src/Liip/RMT/Action/BuildPharPackageAction.php

```diff
@@ -0,0 +1,114 @@
+<?php
+/*
+ * This file is part of the project RMT
+ *
+ * Copyright (c) 2013, Liip AG, http://www.liip.ch
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Liip\RMT\Action;
+
+use Phar;
+use FilesystemIterator;
+
+use Liip\RMT\Context;
+
+/**
+ * Builds a Phar package of the current project.
+ */
+class BuildPharPackageAction extends BaseAction
+{
+
+    protected $releaseVersion;
+
+    public function __construct($options)
+    {
+        $this->options = array_merge(array(
+            'package-name' => 'rmt-package',
+            'destination' => '/tmp/',
+            'excluded-paths' => '',
+            'metadata' => array(),
+            'default-stub-cli' => '<?php __HALT_COMPILER(); ?>',
+            'default-stub-web' => '<?php __HALT_COMPILER(); ?>',
```

Comment:
AFAIK, the __HALT_COMPILER() call is the minimal content for the stub. That's why I added it there. I also compared the contents of some generated stubs and the __HALT_COMPILER() call is always the last instruction.
Regarding its function, I agree the docs are not that clear, but it seems to do just that: halting the compiler during the package compilation.